### PR TITLE
Top artist testing

### PR DESCRIPTION
**Pull request by nicktu12**
@ameseee @robbiegreiner @lfinney @francylang

## What's this PR do?

Fix scss preprocessor and implement testing for top artists

## Pre-Merge TODOs
lost of new tests for top artists and update mock data

## Learning

@wvmitchell is pretty much a wizard

## Where should the reviewer start?

most the good shit happens in top artists . test. js

### CODE REVIEW

File: src/TopArtists/TopArtists.test.js [hunk 1]
```diff
@@ -10,29 +10,67 @@ import {
...
+
+    expect(wrapper.state().selected).toEqual(3);
+  }) 
+
+  it('should be able to return state to null if same card is clicked', ()=>{
+    wrapper.instance().addSelected(1);
+    wrapper.instance().addSelected(1);
+
+    expect(wrapper.state().selected).toEqual(null);
+  })
```
Comment (wvmitchell) [inline src/TopArtists/TopArtists.test.js]: Dude, these are excellent. Good job testing around multiple edge cases.

**Comment by wvmitchell** [comment]
👍

**Comment by lfinney** [comment]
Nice looking tests you've written there, Nick. Your mock data is nicely built out, as well.
